packages/cardano-services/test/InMemoryCache/InMemoryCache.test.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { CACHE_TTL_DEFAULT, InMemoryCache } from '../../src/InMemoryCache';
+import NodeCache from 'node-cache';
+
+jest.mock('node-cache');
+
+describe('In-Memory Cache', () => {
```

Comment:
Best to just use the module name here
```suggestion
describe('InMemoryCache', () => {
```